src/Symfony/Component/Serializer/Normalizer/UnwrappingDenormalizer.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Serializer\Normalizer;
+
+use Symfony\Component\PropertyAccess\PropertyAccess;
+use Symfony\Component\Serializer\Exception\NotNormalizableValueException;
+use Symfony\Component\Serializer\SerializerAwareInterface;
+use Symfony\Component\Serializer\SerializerInterface;
+
+class UnwrappingDenormalizer implements DenormalizerInterface, SerializerAwareInterface
```

Comment:
```suggestion
final class UnwrappingDenormalizer implements DenormalizerInterface, SerializerAwareInterface
```

And can't this normalizer implements https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Serializer/Normalizer/CacheableSupportsMethodInterface.php ?